app/src/main/java/com/android/brewr/ui/overview/AddJourney.kt

```diff
@@ -150,46 +149,26 @@ fun AddJourneyScreen(
               // Date
 
               var selectedDate by remember { mutableStateOf(date) }
-              DateField(selectedDate) { selectedDate = it }
+              DateField(null) { selectedDate = it }
```

Comment:
Your new code seems amazing and work perfectly, but I have a question about this part.
As null is set as the initial value, the user see the message saying that the date is not selected. But actually it is selected and saved as Timestamp.now even when the user does not choose the date. 
So I was wondering why did you choose null as an initial value instead of selectedDate here. I think showing the current date could be less confusing because it is saved as Timestamp.now when the user does not choose the date.

Reply:
I agree!! Should we set the current date as the default?

Reply:
I think that will be good!!